Firestore/core/test/unit/local/local_serializer_test.cc

```diff
@@ -494,6 +497,42 @@ TEST_F(LocalSerializerTest, EncodesTargetData) {
   ExpectRoundTrip(target_data, expected);
 }
 
+TEST_F(LocalSerializerTest, EncodesTargetDataWillDropExpectedCount) {
+  core::Query query = Query("room");
+  TargetId target_id = 42;
+  ListenSequenceNumber sequence_number = 10;
+  SnapshotVersion version = testutil::Version(1039);
+  SnapshotVersion limbo_free_version = testutil::Version(1000);
+  ByteString resume_token = testutil::ResumeToken(1039);
+
+  TargetData target_data(query.ToTarget(), target_id, sequence_number,
+                         QueryPurpose::Listen, SnapshotVersion(version),
+                         SnapshotVersion(limbo_free_version),
+                         ByteString(resume_token), 1234);
```

Comment:
nit: Add the `/*expected_count=*` comment here in the parameter list to the TargetData constructor. Also on line 601.